api/src/Entity/MaterialList.php

```diff
@@ -117,6 +138,6 @@ public function copyFromPrototype($prototype, $entityMap): void {
         $entityMap->add($prototype, $this);
 
         $this->materialListPrototypeId = $prototype->getId();
-        $this->name = $prototype->name;
+        $this->name = $prototype->getName();
```

Comment:
Ich würde hier bei `$prototype->name` bleiben.
Ich glaube hier sollte nicht die Anzeige-Logik in gespeicherte Daten übernommen werden.
(z.B. "NoName")

Ebenfalls als "good first issue" machbar.

Reply:
Bin nicht sicher ob ich hier zustimme.
In Lager-Vorlagen rechne ich nicht damit, dass viele CampCollaboration-Materiallisten vorhanden sind, im Optimalfall sogar keine.
Falls wir in Zukunft mal das "klonen" von normalen Lagern erlauben, fände ich es aber besser, wenn ich z.B. noch sehe, welches Material letztes Jahr von wem genau organisiert wurde, statt dass dann fünf "NoName" Materiallisten im Lager sind. Grund ist, dass ich bei der diesjährigen Durchführung möglicherweise diese Leute anfragen möchte, ob sie vorbereitetes Material noch bei sich haben (z.B. digitale Ausbildungsunterlagen).

Reply:
In https://github.com/ecamp/ecamp3/issues/2574